core/src/main/java/org/apache/accumulo/core/conf/SiteConfiguration.java

```diff
@@ -80,12 +93,28 @@ private SiteConfiguration() {
     staticConfigs = Collections.unmodifiableMap(temp);
   }
 
+  private static URL toURL(File f) {
+    try {
+      return f.toURI().toURL();
+    } catch (MalformedURLException e) {
+      throw new IllegalStateException(e);
```

Comment:
If it's a malformed URL, then it's an illegal argument to the method, rather than an illegal state.

Reply:
Fixed now